src/components/step/ConnectDeviceAccountStepContainer/ConnectDeviceAccountStepContainer.tsx

```diff
@@ -2,13 +2,34 @@ import React, { useState, useEffect } from "react";
 import MyDataHelps, { StepConfiguration } from "@careevolution/mydatahelps-js";
 import ConnectDeviceAccountStep from "../ConnectDeviceAccountStep";
 
-export default function () {
+export interface ConnectDeviceAccountStepContainerProps {
+    deviceType?: string;
+    providerID?: number;
+}
+
+export default function (props: ConnectDeviceAccountStepContainerProps) {
     const [title, setTitle] = useState<string>();
     const [text, setText] = useState<string>();
     const [styles, setStyles] = useState<any>({});
-    const [deviceType, setDeviceType] = useState<string>("");
-    const [providerID, setProviderID] = useState<number>();
-    const [connected, setConnected] = useState<boolean>();
+    const [deviceType, setDeviceType] = useState<string>(
+        props.deviceType ?? ""
+    );
+    const [providerID, setProviderID] = useState<number>(props.providerID ?? 0);
```

Comment:
Would drop the state param and prop since it's not needed, you can just get the providerID with a method